tests/driver_si1145/main.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2016 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the SI1145 UV, IR, visible and proximity
+ *              sensor.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#ifndef TEST_I2C
+#error "TEST_I2C not defined"
+#endif
+
+#include <stdio.h>
+#include <inttypes.h>
+
+#include "si1145.h"
+#include "xtimer.h"
+#include "board.h"
+
+#define SLEEP_2S   (2 * 1000 * 1000u) /* 2 seconds delay between printf */
+
+int main(void)
+{
+    si1145_t dev;
+    uint16_t uv = 0, ir = 0, visible = 0, distance = 0;
+    int result;
+
+    puts("SI1145 test application\n");
+
+    printf("+------------Initializing------------+\n");
+    result = si1145_init(&dev, TEST_I2C);
+    if (result == -1) {
+        puts("[Error] The given i2c is not enabled");
```

Comment:
Actually, the sensor was not found :-)

I would say: [Error] Unable to communicate with sensor.